### PR TITLE
Added descriptions for frontend projects in data.json

### PR DESCRIPTION
**Pull request by Mayank-704**
Closes: #1046

- **Title:** Added descriptions for frontend projects in `data.json`
- **Name:** Mayank Gupta
- **Idenitfy yourself:** KWOC Contributor 

### Describe the add-ons or changes you've made 📃
I added detailed descriptions for the frontend projects in the `data.json` file as part of the open-source contribution. These descriptions provide better clarity for the frontend project entries.
A sample:
- "description": "The hotstar-clone project is a web page that replicates the interface of the popular streaming platform Hotstar using HTML5, CSS3, and JavaScript. This project focuses on creating a responsive design with interactive features such as navigation, content highlights, and media sections. It’s a valuable project for practicing the integration of JavaScript with HTML and CSS to enhance interactivity and user engagement."

### Screenshots 📷

screenshot webp

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Descriptions have been added in the `src/database/data.json` file for frontend projects. Please review and let me know if any adjustments are needed.

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
- Resolve merge conflicts
- Link your issue properly
- Don't change indentation, only commit your work

**Comment by Mayank-704** [comment]
@Avdhesh-Varshney Do I need to close this PR and make a new one with single commit or I can do one more commit here for resolving merge conflict.

**Comment by Avdhesh-Varshney** [comment]
> @Avdhesh-Varshney Do I need to close this PR and make a new one with single commit or I can do one more commit here for resolving merge conflict.

Yes, work in this pr

**Comment by Mayank-704** [comment]
@Avdhesh-Varshney  Done 
- Resolved Merge Conflict (Didn't do linting)
- Linked Issue Properly

File: src/database/data.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
 				"date": "Sat Nov 02 2024",
 				"github_url": "https://github.com/Suvadip-sana",
 				"avatar_url": "https://avatars.githubusercontent.com/u/78638404?v=4",
-				"description": "No description found",
+        "description": "This project is a frontend replica of the popular e-commerce platform Amazon. It demonstrates key web development skills by implementing a visually appealing and responsive user interface. Built using HTML5 and CSS3, the clone showcases features like a dynamic product display, categories, and search functionality, offering a hands-on experience with creating a scalable and intuitive e-commerce website. This project is ideal for learners looking to deepen their understanding of UI/UX design, layout structuring, and styling techniques in web development. It’s also a great portfolio piece for showcasing expertise in frontend development.",
```
Comment (Avdhesh-Varshney) [inline src/database/data.json:81]: fix the indentation of description

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Pull request closed without merging**